### DISCUSSION TITLE
忽略mac自带的触摸板, "自动切换到活跃设备"只作用于不同鼠标

### DISCUSSION THREAD

**Discussion by luistrong** [linearmouse/linearmouse#597] [Ideas]
如果勾选"自动切换到活跃设备", 使用触摸板时就会切换到触摸板, 导致滑动很怪异. 但由于有好几个鼠标设备, 如果不勾选"自动切换到活跃设备", 每次都要切换鼠标, 不太方便.
能否实现只针对不同鼠标之间自动切换, 而忽略自带触摸板呢? 谢谢!

**Comment by lujjjh** [comment]
我可能没有完全理解你的意思。LinearMouse 支持针对不同设备应用不同的配置，“自动切换到活跃设备”的作用是在 LinearMouse 设置界面中自动切换到当前活跃的设备，免去手动选择要配置的设备的麻烦。这个选项的开启或者关闭不会影响 LinearMouse 运行时的其他行为。

> 如果勾选"自动切换到活跃设备", 使用触摸板时就会切换到触摸板, 导致滑动很怪异

是指在配置鼠标的设置时不小心触碰到触摸板，错误地配置成了触摸板吗？

**Reply by ru-wang** [reply]
> 是指在配置鼠标的设置时不小心触碰到触摸板，错误地配置成了触摸板吗？

没错 我偶尔也会有这种情况 不过我只有一个鼠标 所以我是会关闭自动切换的
